docs/zh_cn/design/infer_translation.md

```diff
@@ -0,0 +1,244 @@
+# Inference interface
+
+When developing based on MMEngine, we usually define a configuration file for a specific algorithm, use the configuration file to build an [executor](./runner.md), and execute the training and testing processes, and save the trained weights. When performing inference based on the trained model, the following steps are usually required:
+
+1. Build the model based on the configuration file
+2. Load the model weights
+3. Set up the data preprocessing pipeline
+4. Perform forward inference on the model
+5. Visualize the inference results
+6. Return the inference results
+
+For this type of standard inference workflow, MMEngine provides a unified inference interface and recommends that users develop inference code based on this interface specification.
+
+## Example Usage
+
+### Defining an Inferencer
+
+Implement a custom `inferencer` based on `BaseInferencer` 
+
+```python
+from mmengine.infer import BaseInferencer
+
+class CustomInferencer(BaseInferencer)
+    ...
+```
+
+For specific details, please refer to the [Development Specification](#Development-Specification-of-Inference-Interface).
+
+### Building an Inferencer
+
+**Building an Inferencer Based on Configuration File**
+
+```python
+cfg = 'path/to/config.py'
+weight = 'path/to/weight.pth'
+
+inferencer = CustomInferencer(model=cfg, weight=weight)
+```
+
+**Building an Inferencer Based on Configuration Class Instance.**
+
+```python
+from mmengine import Config
+
+cfg = Config.fromfile('path/to/config.py')
+weight = 'path/to/weight.pth'
+
+inferencer = CustomInferencer(model=cfg, weight=weight)
+```
+
+**Building an Inferencer based on model name defined in 'model - index**. Take the [ATSS detector in MMDetection](https://github.com/open-mmlab/mmdetection/blob/31c84958f54287a8be2b99cbf87a6dcf12e57753/configs/atss/metafile.yml#L22) as an example, the model name is `atss_r50_fpn_1x_coco`, since the weight path is already defined in the model-index, the weight parameter does not need to be configured.
```

Comment:
```suggestion
**Building an Inferencer based on model name defined in model-index**. Take the [ATSS detector in MMDetection](https://github.com/open-mmlab/mmdetection/blob/31c84958f54287a8be2b99cbf87a6dcf12e57753/configs/atss/metafile.yml#L22) as an example, the model name is `atss_r50_fpn_1x_coco`. Since the path of weight has already been defined in the model-index, we don't need to configure the weight arguments anymore.
```